skills/csharp/experimental/itsmskill/Dialogs/Teams/TicketTaskModule/DeleteTicketTeamsImplementation.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ITSMSkill.Dialogs.Teams.TicketTaskModule
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using ITSMSkill.Extensions.Teams;
+    using ITSMSkill.Models;
+    using ITSMSkill.Services;
+    using ITSMSkill.TeamsChannels;
+    using ITSMSkill.TeamsChannels.Invoke;
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Schema.Teams;
+    using Microsoft.Extensions.DependencyInjection;
+
+    /// <summary>
+    /// DeleteTicketTeamsImplementation for Deleting a Ticket Handles OnFetch and OnSumbit Activity for TaskModules
+    /// </summary>
+    [TeamsInvoke(FlowType = nameof(TeamsFlowType.DeleteTicket_Form))]
```

Comment:
doesn't seem like this class has actual content. are you planning on adding more into this class in the future?

Reply:
removed the classes as i implemented them in another PR.